include/swift/Parse/ParseSILSupport.h

```diff
@@ -13,11 +13,10 @@
 #ifndef SWIFT_PARSER_PARSESILSUPPORT_H
 #define SWIFT_PARSER_PARSESILSUPPORT_H
 
+#include "swift/SIL/SILParser.h"
```

Comment:
Why the explicit header dependency instead of a forward declaration of `ReadSIL`?

Reply:
Good point, I'll fix it.